src/main/java/org/broadinstitute/consent/http/service/DarCollectionService.java

```diff
@@ -464,46 +463,71 @@ public List<Integer> findDatasetIdsByDACUser(User user) {
   }
 
 
-  public DarCollection getByReferenceId(String referenceId) {
+  public DarCollection getByReferenceId(User user, String referenceId) {
     DarCollection collection = darCollectionDAO.findDARCollectionByReferenceId(referenceId);
     if (Objects.isNull(collection)) {
       throw new NotFoundException(
           "Collection with the reference id of " + referenceId + " was not found");
     }
-    return addDatasetsToCollection(collection);
+    return filterCollectionVotesForUser(user, addDatasetsToCollection(collection));
   }
 
-  public DarCollection getByCollectionId(Integer collectionId) {
+  public DarCollection getByCollectionId(User user, Integer collectionId) {
     DarCollection collection = darCollectionDAO.findDARCollectionByCollectionId(collectionId);
     if (Objects.isNull(collection)) {
       throw new NotFoundException(
           "Collection with the collection id of " + collectionId + " was not found");
     }
-    return addDatasetsToCollection(collection);
+    return filterCollectionVotesForUser(user, addDatasetsToCollection(collection));
   }
 
-  public DarCollection getCollectionWithAllElectionsByCollectionId(Integer collectionId) {
-    DarCollection collection = darCollectionDAO.findCollectionWithAllElectionsByCollectionId(collectionId);
+  /**
+   * Given a DarCollection, remove all votes from elections for roles that should not see them. This
+   * method mutates the given collection.
+   *
+   * @param user       The User requesting the collection
+   * @param collection DarCollection to filter votes from
+   * @return DarCollection
+   */
+  private DarCollection filterCollectionVotesForUser(User user, DarCollection collection) {
+    // Individual votes are only visible to CHAIRPERSON, MEMBER, and ADMIN roles
+    List<UserRoles> voteViewRoles = List.of(UserRoles.CHAIRPERSON, UserRoles.MEMBER,
```

Comment:
Are members supposed to see how other members voted?  My understanding was only Chairs and Admins were allowed to see how everyone in a DAC voted.

Reply:
Follow-ups from conversation:
1. We do, in fact, currently show individual member votes to other members so we need to keep them in the results entity.
2. SOs do not currently have a view of votes. Until we need that functionality, we can keep it restricted to Admins, Chairs, and Members.